src/Swift.Runtime/src/Swift/SwiftString.cs

```diff
@@ -66,29 +69,30 @@ protected virtual void Dispose(bool disposing)
 
     public static nuint PayloadSize => _payloadSize;
 
-    public Buffer Payload => _payload;
+    public TypeBuffer Buffer => _buffer;
```

Comment:
Nit one: `TypeBuffer` as a name - I don't understand the "Type" part - but it's possible I'm not versed in the terminology.
Nit two: `TypeBuffer`'s fields are public - so this is effectively exposing the internal implementation details of Swift - do we really want that? For all other purposes we treat the string's storage as an opaque buffer...

Reply:
Nit one: Tried to avoid potential mistakes with System.Buffer. Reverted name and prefixed with parent type name.
Nit two: Fixed.